dub.sdl

```diff
@@ -1,6 +1,6 @@
 name "jsoniopipe"
 description "JSON and JSON5 parser for iopipe"
-authors "Steven Schveighoffer"
+authors "Steven Schveighoffer, Gaofei Qiu"
```

Comment:
Remove this, it will just conflict with the other PR when it is merged.

Reply:
Well, I guess it doesn't conflict, but I don't understand why. I guess because it's the same text?